accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -944,6 +944,19 @@ public CommandsBuilder<State, SystemUnderTest> addIf(Predicate<State> predicate,
             return this;
         }
 
+        @SafeVarargs
+        public final CommandsBuilder<State, SystemUnderTest> addAllIf(Predicate<State> predicate,
```

Comment:
To me, `addAllIf` is hard to understand. My mental model when processing it reads as 
```
if (predicate) 
{
   first;
   second;
   rest...;
}
```
So, maybe just `addIf`. 
I guess you mean "add all the if statements". Since they are using the same predicate, it is really one single `if` statement.